letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +155,37 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    @mock.patch("acme.challenges.TLSSNI01Response.simple_verify")
+    def test_simple_verify(self, mock_http01_verify, mock_tlssni01_verify):
+        # pylint: disable=protected-access
```

Comment:
Still, please move the comment next to the `_solve_challenges` line, or even make it `self.handler._solve_challenges()  # pylint: disable=protected-access` if it is under 80 chars.